.github/workflows/release.yaml

```diff
@@ -65,21 +65,22 @@ jobs:
           gsutil -h "Cache-Control:no-cache,max-age=0" cp build/helm/${version}.tgz gs://helm.coder.com/logstream-kube
           gsutil -h "Cache-Control:no-cache,max-age=0" cp build/helm/index.yaml gs://helm.coder.com/logstream-kube
 
-      - name: Create Release
-        uses: actions/create-release@v1
-        id: create_release
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        with:
-          release_name: ${{ steps.version.outputs.version }}
-          tag_name: ${{ github.ref }}
+      - name: Create and upload release
+        run: |
+          set -euo pipefail
+          version=${{ steps.version.outputs.version }}
 
-      - name: Upload Helm Release Asset
-        uses: actions/upload-release-asset@v1
+          if gh release view $version; then
+            echo "Release $version already exists"
+            exit 0
+          else
```

Comment:
nit: can remove else case and outdent below block due to `exit 0`

Reply:
I also discovered that the `gh release view` is interactive and ensured it could run noninteractively.